docs/CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ Entries inside each section should be ordered by type:
 ## CLI
 !-->
 
+# unreleased - YYYY-MM-DD
+
+## Python API
+
+* [Changed] Add `catalog` fragment to Quilt+ URIs (and to documentation). ([#4213](https://github.com/quiltdata/quilt/pull/4213))
```

Comment:
how is this related to the python api?

Reply:
Doh! Dumb copy-and-paste error.

Reply:
we have a separate changelog for the catalog